@navikt/core/css/layout-container.css

```diff
@@ -0,0 +1,28 @@
+.navds-layout-container {
+  padding-left: var(--ac-l-container-padding-xs, var(--navds-spacing-4));
+  padding-right: var(--ac-l-container-padding-xs, var(--navds-spacing-4));
+  margin-left: auto;
+  margin-right: auto;
+  max-width: var(--a-breakpoint-lg);
+}
+
+@media (min-width: 48rem) {
+  .navds-layout-container {
+    padding-left: var(--ac-l-container-padding-xs, var(--navds-spacing-6));
+    padding-right: var(--ac-l-container-padding-xs, var(--navds-spacing-6));
+  }
+}
+
+@media (min-width: 64rem) {
+  .navds-layout-container {
+    padding-left: var(--ac-l-container-padding-xs, var(--navds-spacing-6));
+    padding-right: var(--ac-l-container-padding-xs, var(--navds-spacing-6));
+  }
+}
+
+@media (min-width: 80rem) {
+  .navds-layout-container {
+    padding-left: var(--ac-l-container-padding-xs, var(--navds-spacing-6));
+    padding-right: var(--ac-l-container-padding-xs, var(--navds-spacing-6));
+  }
+}
```

Comment:
Er det meningen at alle disse skal være like? I så fall kan man droppe 64 og 80, for de utgjør vel ingen praktisk forskjell.

Reply:
Ja, det ser ikke ut til at de trengs.

Reply:
bare glemt å endre til `--ac-l-container-padding-sm/md/lg`

Reply:
Et alternativ til å sette padding på sidene av container er å sette en bredde på X prosent, men det er kanskje ikke like presist eller forutsigbart?

```css
.navds-layout-container {
    width: 90%;
    max-width: 80rem;
}
```

Reply:
Ja, vær på utkikk etter mer "fluid" løsninger: altså løsninger som er responsive uten å bruke brekkpunkter, men som heller bruker ting som relative enheter og flex-wrap etc.

Reply:
Tror kanskje "fluid" kan være en prop/valgmulighet på container da context for bruk kan være veldig varierende. Litt usikker på hva vi ønsker scopet på Container skal være. Bare være en max-width wrapper basert på våre definisjoner på layout? Innholds-wrapper for selve layout på siden? Da må kanskje width/maxwidth ha flere predefinerte verdier.